.ansible-lint

```diff
@@ -0,0 +1,10 @@
+---
+exclude_paths:
+  - molecule/
+  - tests/
+
+skip_list:
+ - fqcn
+ - yaml[line-length]
+ - yaml[empty-lines]
+ - yaml[trailing spaces]
```

Comment:
Please add to ignore list

- yaml[truthy]
- jinja[spacing]

Reply:
This has been updated, I also corrected the typo in `yaml[trailing spaces]` > `yaml[trailing-spaces]`